### DISCUSSION TITLE
administration on feedback channel

### DISCUSSION THREAD

**Discussion by L-Ojijo** [lsc-hubs/kenya-hub#21] [General]
We should have option for edit feedback, delete feedback so that one user does not post multiple feedbacks  or ease of editing fedback
 What do you think?

**Comment by pvgenuchten** [comment]
Administration of contributed content is an important activity in any community, I fully agree @L-Ojijo

This option is available, you can edit your own feedback (at https://github.com/lsc-hubs/kenya-hub/discussions/21) or administrators (like me) can do it

we'll soon start the discussion who will enter the board of administrators
